test/DynamoCoreWpfTests/SerializationTests.cs

```diff
@@ -535,14 +535,20 @@ private void DoWorkspaceOpen(string filePath)
             //no longer do this, as its done in model serialization tests.
             //ConvertCurrentWorkspaceToDesignScriptAndSave(filePathBase);
 
-            string json = saveFunction(this.ViewModel, filePathBase);
+            string json = saveFunction(this.ViewModel, filePathBase, filePath);
 
```

Comment:
I could refactor the original code to remove `filePathBase` and just use `filePath`.  Since I don't know if I am dealing with a dyn or dyf it is more convenient to have the entire path

Reply:
I think I am going to make this change as well since passing `filePathBase` and `filePath` is pretty redundant

Reply:
Agree